app/services/ucas_matching/matching_data_export.rb

```diff
@@ -0,0 +1,109 @@
+module UCASMatching
+  class MatchingDataExport
+    HEADER_NAMES = {
+      apply_candidate_id: 'Apply candidate ID',
+      first_name: 'First Name',
+      surname: 'Surname',
+      dob: 'DOB',
+      address_line_1: 'Address line 1',
+      address_line_2: 'Address line 2',
+      address_line_3: 'Address line 3',
+      address_line_4: 'Address line 4',
+      application_state: 'Application state',
+      country: 'Country',
+      postcode: 'Postcode',
+      email_address: 'Email address',
+      provider_code: 'Provider code',
+      provider_name: 'Provider name',
+      phase: 'Apply stage', # <- what we call our field: how UCAS refer to it
```

Comment:
Late question! If there submitted-but-withdrawn applications in Apply 2 they can share a candidate ID with a "live" Apply 2 application, as in Mike E's Slack message yesterday — https://ukgovernmentdfe.slack.com/archives/CQA64BETU/p1594032853157000. These won't be uniquely identifiable from the combination of Candidate ID and phase... but perhaps that doesn't matter to UCAS?

https://www.apply-for-teacher-training.education.gov.uk/support/applications/2319
https://www.apply-for-teacher-training.education.gov.uk/support/applications/2321

Reply:
I don't know about this, perhaps discuss this some other time? This PR doesn't change the code at all.

Reply:
Sure — no intention of changing anything, just curious